spec.bs

```diff
@@ -3910,9 +3910,14 @@ To <dfn>handle ad auction signals header value</dfn> given a [=byte sequence=] |
   1. Let |parsedSignals| be the result of [=parsing JSON bytes to an Infra value=], given
     |adAuctionSignals|.
   1. If |parsedSignals| is failure or not a [=list=], return.
+  1. Let |headerAdSlots| be a new [=ordered set=].
   1. [=list/For each=] |signal| of |parsedSignals|:
     1. If |signal| is not an [=ordered map=], [=iteration/continue=].
     1. If |signal|["`adSlot`"] doesn't exist, [=iteration/continue=].
+    1. If |headerAdSlots| [=set/contains=] |signal|["`adSlot`"], [=iteration/continue=]. Optionally,
+      [=report a warning to the console=] with a diagnostic error message indicating that duplicate
+      [:Ad-Auction-Signals:] `adSlot` dictionary was ignored.

```

Comment:
Should dictionary be "dictionaries"? Or should it say that "a" duplicate [....] dictionary was ignored? The wording feels a little funky for some reason. Either way, not an LGTM-blocker.